tests/WP_Ultimo/Helpers/Screenshot_Test.php

```
@@ -302,16 +303,37 @@ function () {
+		$log_listener           = function ($handle, $message, $level) use (&$logged_level) {
+			if ('screenshot-generator' === $handle) {
+				$logged_level = $level;
+			}
+		};
+
+		wu_save_setting('error_logging_level', 'all');
+		add_action('wu_log_add', $log_listener, 10, 3);
 		add_filter(
 			'pre_http_request',
-			function () {
+			function ($preempt, $args) use (&$request_args) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$preempt'. (undefined)

(UnusedFormalParameter)